roles/adminconsole/ac_configAc/tasks/main.yaml

```diff
@@ -24,6 +24,7 @@
     recurse: yes
   with_items:
     - "{{ tgt_adminConsole_sec_dir }}/tls/rootca"
+    - "{{ tgt_adminConsole_sec_dir }}/tls/server"
```

Comment:
Can you explain why broker's private key is needed?

The generated private key is unique and *private* to broker. It shouldn't be existing outside the broker. Otherwise, it is going to violate the security policy and the Pulsar cluster won't be secure any more.

AdminConsole, as a "client" connecting to the broker, I don't see why it needs broker's private key.

Reply:
According to https://pulsar.apache.org/docs/security-tls-transport/, when the component is acting as proxy, these information will be needed to for establishing two way communication. So, I asked Chris B whether the new adminconsole as acting the proxy. His answer is "so like it". There is nothing wrong in security to configure two way communication with the same keys between two servers as long as they behind the firewalls and communication only only through limited port with limited protocol. You noticed that because you see the filename in the configuration file.  If you insist, we need to change it, we can change the filename

Reply:
This is not correct. In any case, you can't share a broker's private key with any other component

Reply:
It is not about changing the file name. One private key uniquely represents a server host. You can't share it among different server hosts. If AdminConsole needs its own private key, then you need to create its own private key.

Reply:
I will change 01. script to add private key for adminconsole although I can not agree. The private key can not be shared only when private key contains both uuid type (hardware) information and dynamic (logical) target information (such as IP address). But, I will change the scripts as your request.